internal/cmd/stack/flags.go

```diff
@@ -62,7 +62,7 @@ var flagRunLatest = &cli.BoolFlag{
 
 var flagNoInit = &cli.BoolFlag{
 	Name:  "noinit",
-	Usage: "Indicate whether to skip initialization for a task",
+	Usage: "Indicate whether to skip initialization for a task.  Initializing the workspace before running a task means executing all the pre-initialization hooks as well as running the vendor-specific initialization procedure",
```

Comment:
```suggestion
	Usage: "Indicate whether to skip initialization for a task. Initializing the workspace before running a task means executing all the pre-initialization hooks as well as running the vendor-specific initialization procedure",
```

You got two spaces there

Reply:
Could this actually just say `Any pre-initialization hooks as well the vendor-specific initialization procedure will be skipped"?

Reply:
@tomasmik's wording suggestion sounds more straightforward to me 👍🏻